content/docs/usage/command_line/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Common CLI Tasks
+linktitle: OpenClarity CLI
+weight: 410
+---
+
+## Initiate a Scan Using the CLI
+
+## Reporting Results into File:
+```
+./cli/bin/openclarity-cli scan --config ~/testConf.yaml -o outputfile
```

Comment:
`bin` folder is at root so this should be ´./bin/openclarity-cli scan --config ~/testConf.yaml -o outputfile´

Reply:
This file is outdated anyway, and will be replaced by the contents of `openclarity/docs/command_line.md` soon, so any changes should be made there